File: LostAnimalsAndroid/app/src/main/java/com/pbl/animals/models/Comment.java
package com.pbl.animals.models;

import com.fasterxml.jackson.annotation.JsonFormat;
import com.pbl.animals.models.base.ModelBase;

import java.util.Date;

public class Comment extends ModelBase {
    public User author;
    public long userId;
    public String content;
    public long postId;
    public Post post;
    public Location location;

    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ss")
    public Date seenTime;

    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ss")
    public Date commentTime;
}

File: LostAnimalsAPI/LostAnimalsAPI/Models/Comment.cs
using LostAnimalsAPI.Models.Auth;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace LostAnimalsAPI.Models
{
    public class Comment: ModelBase
    {
        public string Content { get; set; }
        public ApplicationUser Author { get; set; }
        public long PostId { get; set; }
        public Post Post { get; set; }
        public Location Location { get; set; }
        public DateTime SeenTime { get; set; }
        public DateTime CommentTime { get; set; }
    }
}
